### PR TITLE
87947534: Use i18n to override field names

### PR DESCRIPTION
**Pull request by cmc333333**
Specifically, the rwa field.

### CODE REVIEW

**Comment by cmc333333** [comment]
Updated to account for a flaky test -- enforce an order to parallel approvers.

File: app/models/cart.rb [hunk 1]
```diff
@@ -69,7 +69,8 @@ def ordered_awaiting_approvals
   # users with outstanding cart_notification_emails
   def currently_awaiting_approvers
```
Comment (phirefly) [inline app/models/cart.rb]: We can use a spec for this

**Comment by cmc333333** [comment]
Specs have been added

**Comment by phirefly** [comment]
:ship: it

**Pull request merged**